### PR TITLE
convert MainModule.java to kotlin

### PR DESCRIPTION
**Pull request by loicsiret**
Following some instruction on :
https://google.github.io/dagger/faq.html#What do I do instead?
Annotation @Module was missing on the conpanion object. Also as you
wish to have a static provides you need to add a @JVMStatic.

### CODE REVIEW

File: app/src/main/java/me/rakshakhegde/dagger_android_injector/screens/main_screen/MainModule.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package me.rakshakhegde.dagger_android_injector.screens.main_screen
+
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+
+@@Module
```
Comment (digitalbuddha) [inline app/src/main/java/me/rakshakhegde/dagger_android_injector/screens/main_screen/MainModule.kt]: double @ symbol

**Comment by loicsiret** [comment]
Hmmm, It seems that I'm not authorized to see bitrise logs.
Can someone post the check failed stacktrace ?

**Comment by rakshakhegde** [comment]
Bitrise logs:
```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring root project 'src'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find com.android.tools.build:gradle:3.0.0-rc2.
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/build/gradle/3.0.0-rc2/gradle-3.0.0-rc2.pom
         https://jcenter.bintray.com/com/android/tools/build/gradle/3.0.0-rc2/gradle-3.0.0-rc2.jar
     Required by:
         project :
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
* Get more help at https://help.gradle.org
BUILD FAILED in 24s
exit status 1
```

File: build.gradle [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
 		compiler_version = "2.4.0-alpha7"
-		kotlin_version = '1.1.2-3'
+		kotlin_version = '1.1.51'
 	}
 	repositories {
 		jcenter()
 	}
 	dependencies {
-		classpath "com.android.tools.build:gradle:$compiler_version"
+		classpath 'com.android.tools.build:gradle:3.0.0-rc2'
```
Comment (rakshakhegde) [inline build.gradle]: Change it back to $compiler_version

File: build.gradle [hunk 3]
```diff
@@ -3,13 +3,13 @@
 buildscript {
 	ext {
 		compiler_version = "2.4.0-alpha7"
```
Comment (rakshakhegde) [inline build.gradle]: Modify this instead

**Review by rakshakhegde — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (loicsiret) [reply build.gradle]: done

**Comment by rakshakhegde** [comment]
@loicsiret Please check buddybuild logs. For getting the databinding library you might need to add `google()` to repositories under project `build.gradle` too.

**Review by rakshakhegde — APPROVED** [review]

**Comment by loicsiret** [comment]
@rakshakhegde Yes, I'm trying to build on my bitrise account. But I still have issue. As soon it is solved, i will update my pr. Thank you

**Comment by rakshakhegde** [comment]
FWIW, you don't need your own bitrise account. Buddybuild should show the logs to you too.

**Pull request merged**